apps/design/backend/scripts/restore_db_backup.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+BACKUP=${1:-latest.dump}
+PGPASSWORD=design dropdb -U design -h localhost design
+PGPASSWORD=design createdb -U design -h localhost design
```

Comment:
Explicitly dropping the db causes warnings like

```
pg_restore: error: could not execute query: ERROR:  relation "public.elections" does not exist
```

When running `pg_restore` because the dump doesn't assume the db has been dropped. We drop it explicitly to reduce the chance of existing db state causing problems with restore.